.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -66,6 +66,12 @@ build_and_install_qemu() {
                 patch -p1 <"$patch"
         done
 
+	# Replace the default-configs/*-softmmu.mak with the tuned ones
+        echo "cp -af ${GOPATH}/src/${PACKAGING_REPO}/qemu/default-configs/*.mak ${GOPATH}/src/${QEMU_REPO}/default-configs/"
```

Comment:
is this a debug log? this message looks odd to me :smile_cat:

Reply:
I had intended to print the log during the qemu building process instead of hiding the config copy-ing. If you thought it is too verbose, I can remove it.

Reply:
> thanks @justin-he
> 
> I guess this depends-on [kata-containers/packaging#1045](https://github.com/kata-containers/packaging/pull/1045)

Yes, exactly. The statement "if [ -e file]" guarantees no errors even without [kata-containers/packaging#1045](https://github.com/kata-containers/packaging/pull/1045)

Reply:
I'd change the log message to something more user friendly

`overwriting default qemu kconfigs` 

 or 

`copying custom qemu kconfigs`

Reply:
Okay.